app/src/auth.ts

```diff
@@ -88,49 +66,19 @@ idleTracker.on('show', () => {
 export async function refresh({ navigate }: LogoutOptions = { navigate: true }): Promise<void> {
 	const appStore = useAppStore();
 
-	// Allow refresh during initial page load
-	if (firstRefresh) firstRefresh = false;
-	// Skip if not logged in
-	else if (!appStore.authenticated) return;
-
-	// Prevent concurrent refreshes
-	if (isRefreshing) return;
-
-	// Skip refresh if access token is still fresh
-	if (appStore.accessTokenExpiry && Date.now() < appStore.accessTokenExpiry - 10000) {
-		// Set a fresh timeout as it is cleared by idleTracker's idle or hide event
-		clearTimeout(refreshTimeout);
-		refreshTimeout = setTimeout(() => refresh(), appStore.accessTokenExpiry - 10000 - Date.now());
-		return;
-	}
-
-	isRefreshing = true;
+	// Allow refresh during initial page load, skip if not logged in
+	if (!appStore.authenticated) return;
```

Comment:
There's an issue here now, preventing to login via SSO providers, because there's no refresh call after being redirect back from the provider.